.github/workflows/commitlint.yml

```
@@ -30,9 +30,8 @@ jobs:
 
       - name: Download configuration if needed
         if: steps.check.outputs.need == 'yes'
-        uses: wei/wget@v1
-        with:
-          args: -O commitlint.config.js https://raw.githubusercontent.com/openedx/edx-lint/master/edx_lint/files/commitlint.config.js
+        run: |
+          wget -O commitlint.config.js https://raw.githubusercontent.com/openedx/edx-lint/HEAD/edx_lint/files/commitlint.config.js
```

If we want identical behavior, we can add the no-verbose option like the dependency did (which is probably desirable):
```suggestion
          wget --no-verbose -O commitlint.config.js https://raw.githubusercontent.com/openedx/edx-lint/HEAD/edx_lint/files/commitlint.config.js
```

---

Good idea, done.